AndroidCharts/src/main/java/im/dacer/androidcharts/LineView.java

```diff
@@ -536,12 +536,12 @@ private Dot findPointAt(int x, int y) {
         int x;
         float y;
         float data;
-        int targetX;
+        double targetX;
         float targetY;
         int linenumber;
         int velocity = MyUtils.dip2px(getContext(), 18);
 
-        Dot(int x, float y, int targetX, float targetY, float data, int linenumber) {
+        Dot(double x, float y, double targetX, float targetY, float data, int linenumber) {
             this.x = x;
```

Comment:
Hi, did you try out your changes? I was about to add them to my fork of the library, as unfortunately this repository seems unmaintained, but your commit doesn't compile for me. For example, `this.x` is an `int` while the local `x` is a double, causing the assignment to fail. Other type-casting errors are also showing.